code/modules/mob/mob_helpers.dm

```diff
@@ -142,7 +142,12 @@ proc/isembryo(A)
 		return 1
 	return 0
 
-proc/isobserver(A)

```

Comment:
OH YOU ADDED SOMETHING BETWEEN TWO LINES?! HERE LET ME JUST SHOW YOU DELETED THE WHOLE THING!